sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -154,6 +179,95 @@ public EditEventSteps(
           webDriverHelpers.waitUntilElementIsVisibleAndClickable(EVENT_DATA_SAVED_MESSAGE);
         });
 
+    When(
+        "I check CLUSTER option on edit Event page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_CLUSTER_EDIT);
+        });
+
+    When(
+        "I select {} option from Primary Mode Of Transmission Combobox on edit Event page",
+        (String option) -> {
+          webDriverHelpers.selectFromCombobox(PRIMARY_MODE_OF_TRANSMISSION_COMBOBOX, option);
+        });
+
+    When(
+        "I click on Epidemiological evidence with ([^\"]*) option",
+        (String option) -> {
+          webDriverHelpers.clickWebElementByText(EPIDEMIOLOGICAL_EVIDENCE_OPTIONS, option);
+        });
+
+    When(
+        "I thick all options for Study on Epidemiological evidence for De version",
```

Comment:
If by "thick" you mean marking a checkbox, then it's a typo (should be "tick", not "thick").